appveyor.yml

```diff
@@ -28,6 +32,25 @@ after_build:
 - cmd: '"C:\Program Files (x86)\CMake\bin\cpack.exe"'
 - ps: |
     Write-Output Get-FileHash SuperTux-*
+    
+    function transfer ($filename)
+    {
+        $file = Get-Item $filename;
+        invoke-webrequest -method put -infile $file.FullName https://transfer.sh/$($file.Name)
```

Comment:
Do we need to upload this to transfer.sh or can we link to AppVeyor, which keeps the builds for longer? If we do need to use transfer.sh, please add notes to the concerned files on the page that they are only available for 2 weeks (or remove them).